help_data.yaml

```diff
@@ -76,3 +66,25 @@ livepatch:
       livepatches cannot be enabled on fips-enabled systems. You can find out
       more about Ubuntu Kernel Livepatch service at
       https://ubuntu.com/security/livepatch
+
+ros:
+  help: |
+    ros provides access to a private PPA which includes available high
+    and critical CVE fixes for Ubuntu Robot Operating System (ROS) packages
+    between the end of the standard Ubuntu LTS security maintenance and its
+    end of life. For full access to ROS ESM and security fixes, both
+    esm-infra and esm-apps services will also be enabled. To get additional
+    bug fixes, enable ros-updates in addition to ros.
+    You can find out more about the ROS ESM service at
+    https://ubuntu.com/robotics/ros-esm
```

Comment:
```suggestion
    ros provides access to a private PPA which includes security-related updates
    for available high and critical CVE fixes for Robot Operating System (ROS) packages
    between the end of the standard Ubuntu LTS security maintenance and its
    end of life. For access to ROS ESM and security updates, both
    esm-infra and esm-apps services will also be enabled. To get additional
    non-security updates, enable ros-updates in addition to ros.
    You can find out more about the ROS ESM service at
    https://ubuntu.com/robotics/ros-esm
```

Reply:
I also find the following difficult to understand, it could use some rewording:
`between the end of the standard Ubuntu LTS security maintenance and its end of life.`